frontend/src/metabase/setup/components/EmbeddingSetup/steps/DoneStep.tsx

```diff
@@ -42,7 +42,7 @@ export const DoneStep = () => {
           description={t`Evolve these starter dashboards or create new analysis.`}
         />
       </Stack>
-      <Button component={Link} to="/" variant="filled">
+      <Button component={Link} role="button" to="/" variant="filled">
```

Comment:
Without `role="button"` we can't use `cy.button` which is weird because in the UI it renders as a button.